prisma/seed.ts

```diff
@@ -23,6 +25,118 @@ async function seed() {
     },
   });
 
+  await prisma.record.createMany({
+    data: [
+      // A record
+      {
+        id: 1,
```

Comment:
Based on https://github.com/Seneca-CDOT/starchart/blob/main/prisma/schema.prisma#L23, you shouldn't need to provide an `id`, it should automatically assign a new one.

Reply:
Based on the call today, sounds like you want this for foreign key.  Resolving.